SimpleMapboxNavigationKt Exception in onDidFinishLoadingStyle

Running `SimpleMapboxNavigationKt` (`master`) produces the following crash 💥 

```
08-24 16:29:57.833 12458-12458/com.mapbox.navigation.examples E/Mbgl-MapChangeReceiver: Exception in onDidFinishLoadingStyle
    android.content.res.Resources$NotFoundException: File res/drawable/mapbox_ic_map_marker_light.xml from drawable resource ID #0x7f0800d6
        at android.content.res.Resources.loadDrawableForCookie(Resources.java:2640)
        at android.content.res.Resources.loadDrawable(Resources.java:2540)
        at android.content.res.Resources.getDrawable(Resources.java:806)
        at android.content.Context.getDrawable(Context.java:458)
        at androidx.core.content.ContextCompat.getDrawable(ContextCompat.java:454)
        at com.mapbox.navigation.ui.internal.ThemeSwitcher.retrieveThemeMapMarker(ThemeSwitcher.java:46)
        at com.mapbox.navigation.ui.map.NavigationMapboxMap.initializeNavigationSymbolManager(NavigationMapboxMap.java:884)
        at com.mapbox.navigation.ui.map.NavigationMapboxMap.<init>(NavigationMapboxMap.java:181)
        at com.mapbox.navigation.ui.map.NavigationMapboxMap.<init>(NavigationMapboxMap.java:140)
        at com.mapbox.navigation.examples.core.SimpleMapboxNavigationKt$onMapReady$2.onStyleLoaded(SimpleMapboxNavigationKt.kt:186)
        at com.mapbox.mapboxsdk.maps.MapboxMap.notifyStyleLoaded(MapboxMap.java:963)
        at com.mapbox.mapboxsdk.maps.MapboxMap.onFinishLoadingStyle(MapboxMap.java:225)
        at com.mapbox.mapboxsdk.maps.MapView$MapCallback.onDidFinishLoadingStyle(MapView.java:1344)
        at com.mapbox.mapboxsdk.maps.MapChangeReceiver.onDidFinishLoadingStyle(MapChangeReceiver.java:198)
        at com.mapbox.mapboxsdk.maps.NativeMapView.onDidFinishLoadingStyle(NativeMapView.java:1159)
        at android.os.MessageQueue.nativePollOnce(Native Method)
        at android.os.MessageQueue.next(MessageQueue.java:323)
        at android.os.Looper.loop(Looper.java:135)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: android.content.res.Resources$NotFoundException: File res/drawable/$mapbox_ic_map_marker_light__0.xml from color state list resource ID #0x7f080008
        at android.content.res.Resources.loadColorStateListForCookie(Resources.java:2750)
        at android.content.res.Resources.loadColorStateList(Resources.java:2699)
        at android.content.res.TypedArray.getColor(TypedArray.java:439)
        at android.graphics.drawable.VectorDrawable$VFullPath.updateStateFromTypedArray(VectorDrawable.java:1605)
        at android.graphics.drawable.VectorDrawable$VFullPath.inflate(VectorDrawable.java:1584)
        at android.graphics.drawable.VectorDrawable.inflateInternal(VectorDrawable.java:666)
        at android.graphics.drawable.VectorDrawable.inflate(VectorDrawable.java:571)
        at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:1215)
        at android.graphics.drawable.Drawable.createFromXml(Drawable.java:1124)
        at android.content.res.Resources.loadDrawableForCookie(Resources.java:2630)
        at android.content.res.Resources.loadDrawable(Resources.java:2540) 
        at android.content.res.Resources.getDrawable(Resources.java:806) 
        at android.content.Context.getDrawable(Context.java:458) 
        at androidx.core.content.ContextCompat.getDrawable(ContextCompat.java:454) 
        at com.mapbox.navigation.ui.internal.ThemeSwitcher.retrieveThemeMapMarker(ThemeSwitcher.java:46) 
        at com.mapbox.navigation.ui.map.NavigationMapboxMap.initializeNavigationSymbolManager(NavigationMapboxMap.java:884) 
        at com.mapbox.navigation.ui.map.NavigationMapboxMap.<init>(NavigationMapboxMap.java:181) 
        at com.mapbox.navigation.ui.map.NavigationMapboxMap.<init>(NavigationMapboxMap.java:140) 
        at com.mapbox.navigation.examples.core.SimpleMapboxNavigationKt$onMapReady$2.onStyleLoaded(SimpleMapboxNavigationKt.kt:186) 
        at com.mapbox.mapboxsdk.maps.MapboxMap.notifyStyleLoaded(MapboxMap.java:963) 
        at com.mapbox.mapboxsdk.maps.MapboxMap.onFinishLoadingStyle(MapboxMap.java:225) 
        at com.mapbox.mapboxsdk.maps.MapView$MapCallback.onDidFinishLoadingStyle(MapView.java:1344) 
        at com.mapbox.mapboxsdk.maps.MapChangeReceiver.onDidFinishLoadingStyle(MapChangeReceiver.java:198) 
        at com.mapbox.mapboxsdk.maps.NativeMapView.onDidFinishLoadingStyle(NativeMapView.java:1159) 
        at android.os.MessageQueue.nativePollOnce(Native Method) 
        at android.os.MessageQueue.next(MessageQueue.java:323) 
        at android.os.Looper.loop(Looper.java:135) 
        at android.app.ActivityThread.main(ActivityThread.java:5417) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
     Caused by: org.xmlpull.v1.XmlPullParserException: Binary XML file line #28: invalid color state list tag gradient
        at android.content.res.ColorStateList.createFromXmlInner(ColorStateList.java:217)
        at android.content.res.ColorStateList.createFromXml(ColorStateList.java:201)
        at android.content.res.Resources.loadColorStateListForCookie(Resources.java:2746)
        at android.content.res.Resources.loadColorStateList(Resources.java:2699) 
        at android.content.res.TypedArray.getColor(TypedArray.java:439) 
        at android.graphics.drawable.VectorDrawable$VFullPath.updateStateFromTypedArray(VectorDrawable.java:1605) 
        at android.graphics.drawable.VectorDrawable$VFullPath.inflate(VectorDrawable.java:1584) 
        at android.graphics.drawable.VectorDrawable.inflateInternal(VectorDrawable.java:666) 
        at android.graphics.drawable.VectorDrawable.inflate(VectorDrawable.java:571) 
        at android.graphics.drawable.Drawable.createFromXmlInner(Drawable.java:1215) 
        at android.graphics.drawable.Drawable.createFromXml(Drawable.java:1124) 
        at android.content.res.Resources.loadDrawableForCookie(Resources.java:2630) 
        at android.content.res.Resources.loadDrawable(Resources.java:2540) 
        at android.content.res.Resources.getDrawable(Resources.java:806) 
        at android.content.Context.getDrawable(Context.java:458) 
        at androidx.core.content.ContextCompat.getDrawable(ContextCompat.java:454) 
        at com.mapbox.navigation.ui.internal.ThemeSwitcher.retrieveThemeMapMarker(ThemeSwitcher.java:46) 
        at com.mapbox.navigation.ui.map.NavigationMapboxMap.initializeNavigationSymbolManager(NavigationMapboxMap.java:884) 
        at com.mapbox.navigation.ui.map.NavigationMapboxMap.<init>(NavigationMapboxMap.java:181) 
        at com.mapbox.navigation.ui.map.NavigationMapboxMap.<init>(NavigationMapboxMap.java:140) 
        at com.mapbox.navigation.examples.core.SimpleMapboxNavigationKt$onMapReady$2.onStyleLoaded(SimpleMapboxNavigationKt.kt:186) 
        at com.mapbox.mapboxsdk.maps.MapboxMap.notifyStyleLoaded(MapboxMap.java:963) 
        at com.mapbox.mapboxsdk.maps.MapboxMap.onFinishLoadingStyle(MapboxMap.java:225) 
        at com.mapbox.mapboxsdk.maps.MapView$MapCallback.onDidFinishLoadingStyle(MapView.java:1344) 
        at com.mapbox.mapboxsdk.maps.MapChangeReceiver.onDidFinishLoadingStyle(MapChangeReceiver.java:198) 
        at com.mapbox.mapboxsdk.maps.NativeMapView.onDidFinishLoadingStyle(NativeMapView.java:1159) 
        at android.os.MessageQueue.nativePollOnce(Native Method) 
        at android.os.MessageQueue.next(MessageQueue.java:323) 
        at android.os.Looper.loop(Looper.java:135) 
        at android.app.ActivityThread.main(ActivityThread.java:5417) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

Regression from https://github.com/mapbox/mapbox-navigation-android/pull/3487

cc @JunDai

--- a/libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/ThemeSwitcher.java
+++ b/libnavigation-ui/src/main/java/com/mapbox/navigation/ui/internal/ThemeSwitcher.java
@@ -9,16 +9,14 @@ import android.graphics.drawable.Drawable;
 import android.preference.PreferenceManager;
 import android.util.AttributeSet;
 import android.util.TypedValue;
-
 import androidx.annotation.AnyRes;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
-import androidx.core.content.ContextCompat;
-
-import com.mapbox.navigation.ui.R;
+import androidx.appcompat.content.res.AppCompatResources;
 import com.mapbox.mapboxsdk.utils.BitmapUtils;
-import com.mapbox.navigation.ui.NavigationView;
 import com.mapbox.navigation.ui.NavigationConstants;
+import com.mapbox.navigation.ui.NavigationView;
+import com.mapbox.navigation.ui.R;
 
 /**
  * This class is used to switch theme colors in {@link NavigationView}.
@@ -43,7 +41,7 @@ public class ThemeSwitcher {
       }
     }
 
-    Drawable markerDrawable = ContextCompat.getDrawable(context, markerResId);
+    Drawable markerDrawable = AppCompatResources.getDrawable(context, markerResId);
     return BitmapUtils.getBitmapFromDrawable(markerDrawable);
   }
 

